controller/instance_manager_controller.go

```diff
@@ -1662,13 +1662,11 @@ func (imc *InstanceManagerController) createInstanceManagerPodSpec(im *longhorn.
 
 func (imc *InstanceManagerController) deleteOrphans(im *longhorn.InstanceManager, isInstanceManagerTerminating bool) error {
 	autoDeletionTypes, err := imc.ds.GetSettingOrphanResourceAutoDeletion()
-	var autoDeleteEngine = false
-	var autoDeleteReplica = false
+	var autoDeleteEnabled = false
 	if err != nil {
 		imc.logger.WithError(err).Warnf("Failed to fetch orphan auto deletion setting, disabled by default")
```

Comment:
Is there a reason we're not returning the error here?

Reply:
No matter the auto deletion is configured, there's still some situation that orphan CR deletion should be initiated:

- instance manager is terminating or terminated
- instance disappears from the instance manager
- the instance is scheduled back to the node, and is no longer considered as orphaned

Reply:
Could you add a comment here?

Reply:
After reviewing again over this function, we should handle the list of orphan CRs belongs to the given instance, not the node of this instance. Will refine this part.

Reply:
> After reviewing again over this function, we should handle the list of orphan CRs belongs to the given instance, not the node of this instance. Will refine this part.

Can you elaborate more on what the handling means?